modules/machineset-migrating-control-plane-nodes-to-diff-sd-rhv.adoc

```diff
@@ -5,7 +5,7 @@
 [id="machineset-migrating-control-plane-nodes-to-diff-sd-rhv_{context}"]
 = Migrating control plane nodes to a different storage domain on {rh-virtualization}
 
-//todo: confirm if this statement is still accurate w/CPMS
+//todo: assess for CPMS impact
```

Comment:
Machine management > Modifying a compute machine set > [Migrating control plane nodes to a different storage domain on RHV](https://53397--docspreview.netlify.app/openshift-enterprise/latest/machine_management/modifying-machineset.html#machineset-migrating-control-plane-nodes-to-diff-sd-rhv_modifying-machineset)

Reply:
I am guessing this is unchanged because it's for RHV?

Reply:
Yeah CPMS has no impact on RHV AFAIK